### PR TITLE
Update summit page with social info

### PR DESCRIPTION
**Pull request by mrbobbytables**
/hold
for review

Adds location details for both the main event and evening social.

### CODE REVIEW

File: content/en/events/kcsna2021/location.md [hunk 1]
```diff
@@ -4,15 +4,45 @@ type: docs
...
 900 West Olympic Boulevard<br>
 Los Angeles, California 90015 USA
 
-More information and an on-site map will be posted closer to the event.
+
+### Lucky Strike
+<img align="right" src="/events/kcsna2021/lucky-strike.jpg" width="30%">
+
+The evening social event will be held at
+<a href="https://www.marriott.com/hotels/travel/laxjw-jw-marriott-los-angeles-la-live/" rel="noopener noreferrer" target="_blank">Lucky Strike</a>,
```
Comment (tokt) [inline content/en/events/kcsna2021/location.md:27]: URL should be: https://www.luckystrikeent.com/locations/los-angeles/ :)

Comment (mrbobbytables) [reply content/en/events/kcsna2021/location.md]: fixed 👍

**Comment by mrbobbytables** [comment]
/hold cancel

good enough for me 👍

**Pull request merged**